keras_tuner/engine/tuner.py

```diff
@@ -31,46 +31,45 @@
-            models. If the hypermodel
-            does not compile the models it generates,
-            then this argument must be specified.
-        metrics: Optional. May be used to override the
-            `metrics` argument in the `compile` step
-            for the models. If the hypermodel
-            does not compile the models it generates,
-            then this argument must be specified.
-        distribution_strategy: Optional. A TensorFlow
-            `tf.distribute` DistributionStrategy instance. If
-            specified, each trial will run under this scope. For
-            example, `tf.distribute.MirroredStrategy(['/gpu:0', '/gpu:1'])`
-            will run each trial on two GPUs. Currently only
-            single-worker strategies are supported.
-        directory: String. Path to the working directory (relative).
-        project_name: Name to use as prefix for files saved
-            by this Tuner.
-        logger: Optional. Instance of Logger class, used for streaming data
-            to Cloud Service for monitoring.
-        tuner_id: Optional. If set, use this value as the id of this Tuner.
-        overwrite: Bool, default `False`. If `False`, reloads an existing project
-            of the same name if one is found. Otherwise, overwrites the project.
+        hypermodel: Instance of HyperModel class (or callable that takes
+            hyperparameters and returns a Model instance).
+        max_model_size: Integer, maximum number of scalars in the parameters of
+            a model. Models larger than this are rejected.
+        optimizer: Optional Optimizer instance.  May be used to override the
+            `optimizer` argument in the `compile` step for the models. If the
+            hypermodel does not compile the models it generates, then this
+            argument must be specified.
+        loss: Optional loss. May be used to override the `loss` argument in the
+            `compile` step for the models. If the hypermodel does not compile
+            the models it generates, then this argument must be specified.
+        metrics: Optional metrics. May be used to override the `metrics`
+            argument in the `compile` step for the models. If the hypermodel
+            does not compile the models it generates, then this argument must
+            be specified.
+        distribution_strategy: Optional instance of `tf.distribute.Strategy`.
+            If specified, each trial will run under this scope. For example,
+            `tf.distribute.MirroredStrategy(['/gpu:0', '/gpu:1'])` will run
+            each trial on two GPUs. Currently only single-worker strategies are
+            supported.
+        directory: A string, the relative path to the working directory.
+        project_name: A string, the name to use as prefix for files saved by
+            this Tuner.
+        logger: Optional instance of Logger class, used for streaming data to
+            Cloud Service for monitoring.
```

Comment:
"used for streaming data to  Cloud Service for monitoring" -> random question: is this still an active feature?

Reply:
I really don't know. I never used this feature. We don't have a tutorial for it. However, from the implementation it is still there. I am not sure if the service of "https://us-central1-kerastuner-prod.cloudfunctions.net/api/" still works or not.